src/server/routes/entity/creator.js

```diff
@@ -139,16 +139,54 @@ const additionalCreatorProps = [
 	'typeId', 'genderId', 'areaId', 'beginDate', 'endDate', 'ended'
 ];
 
-router.post('/create/handler', auth.isAuthenticatedForHandler, (req, res) =>
-	entityRoutes.createEntity(
-		req, res, 'Creator', _.pick(req.body, additionalCreatorProps)
-	)
-);
 
-router.post('/:bbid/edit/handler', auth.isAuthenticatedForHandler, (req, res) =>
-	entityRoutes.editEntity(
+function transformNewForm(data) {
```

Comment:
Is the idea here to make this a temporary shim before we move to better data backend stuff? If so, please comment to that effect.